src/generic/clrpickerg.cpp

```diff
@@ -57,7 +57,9 @@ bool wxGenericColourButton::Create( wxWindow *parent, wxWindowID id,
     // and handle user clicks on it
     Bind(wxEVT_BUTTON, &wxGenericColourButton::OnButtonClick, this, GetId());
 
-    m_bitmap = wxBitmap(FromDIP(defaultBitmapSize));
+    wxBitmap bmp;
+    bmp.CreateWithDIPSize( defaultBitmapSize, GetDPIScaleFactor() );
+    m_bitmap = bmp;
```

Comment:
This could be
```suggestion
    m_bitmap.CreateWithDIPSize( defaultBitmapSize, GetDPIScaleFactor() );
```
or is there something I'm missing here?